ami/trainers/image_vae_trainer.py

```diff
@@ -0,0 +1,56 @@
+from functools import partial
+
+import torch
+from torch.distributions import kl_divergence
+from torch.distributions.normal import Normal
+from torch.nn.functional import mse_loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from ami.data.buffers.buffer_names import BufferNames
+from ami.models.model_names import ModelNames
+from ami.models.model_wrapper import ModelWrapper
+from ami.models.vae import VAE, Decoder, Encoder
+
+from .base_trainer import BaseTrainer
+
+
+class ImageVAETrainer(BaseTrainer):
+    def __init__(
+        self,
+        partial_dataloader: partial[DataLoader[torch.Tensor]],
+        partial_optimizer: partial[Optimizer],
+        device: torch.device,
+    ):
+        super().__init__()
+        self.partial_optimizer = partial_optimizer
+        self.partial_dataloader = partial_dataloader
+        self.device = device
+
+    def on_data_users_dict_attached(self) -> None:
+        self.data_user = self.get_data_user(BufferNames.IMAGE)
```

Comment:
ここ`image_data_user`にしてください。
```suggestion
        self.image_data_user = self.get_data_user(BufferNames.IMAGE)
```

Reply:
https://github.com/MLShukai/ami/pull/75/commits/c668f16fa06ceae56ea13c08794381d4801e43e8
修正しました